### ISSUE TITLE
Add support for attachToAllClusters to dbcUpload

### ISSUE THREAD

**Issue by jakajancar** [databricks/sbt-databricks#19]

**Comment by brkyvz** [comment]
Hi @jakajancar,
You should be able to achieve this already using 
```
dbcClusters := Seq("DBC_ALL_CLUSTERS")
```
Would that help, or do you have a separate use case?

Thanks!

**Comment by jakajancar** [comment]
@brkyvz This checks all clusters (red) as opposed the "all clusters" checkbox (green):

![screenshot](https://dl.dropboxusercontent.com/u/41764/share/20160111/Screenshot%202016-01-11%2014.54.25.png)

**Comment by jakajancar** [comment]
Alternatively, perhaps instead of attachToAllClusters in dbcUpload simply DBC_ALL_CLUSTERS should be changed from "ALL_CLUSTERS" to "__ALL_CLUSTERS" and the special handling removed, and dbcAttach would be used (although an extra unnecessary API call).

**Comment by jakajancar** [comment]
@brkyvz Any thoughts on this?

**Comment by brkyvz** [comment]
Hi @jakajancar,
It would be a simple single line addition. I can make the change that if
```
dbcClusters := Seq("DBC_ALL_CLUSTERS")
```
then we will "click" that green checkbox. I'll submit a PR momentarily.

**Comment by jakajancar** [comment]
OK, let me know when you're done and I can test.

**Comment by brkyvz** [comment]
Resolved with PR #20

**Comment by brkyvz** [comment]
@jakajancar Released v0.1.4!
